packages/office-ui-fabric-react/src/components/Image/Image.Props.ts

```diff
@@ -27,6 +27,12 @@ export interface IImageProps extends React.HTMLProps<HTMLImageElement> {
    * If true, the image frame will expand to fill its parent container.
    */
   maximizeFrame?: boolean;
+
+  /**
+   * Optional callback method for when the image is loaded or is in error.
+   * The 'success' parameter indicates whether the loading was successful or in error.
+   */
+  onImageLoad?: (success: boolean) => void;
```

Comment:
actually after reading the use case, maybe we should have this but rename it as `onLoadingStateChange` and provide the loading state. That way if a new image is piped in you can monitor the loading state. And you can expect that it will definitely be called on every state transition.